### ISSUE TITLE
TreeSelect: ARIA hidden element must not be focusable or contain focusable elements

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#15244]
### Describe the bug

Two empty span element between item wrapper div (p-treeselect-items-wrapper &p-multiselect-items-wrapper) with attribute aria-hidden="true" contains focusable elements. 
firstHiddenFocusableElement and lastHiddenFocusableElement: aria-hidden attribute should be removed.

Same issue with MultiSelect fixed in: https://github.com/primefaces/primeng/pull/15198

### Environment

Chrome

### Reproducer

https://primeng.org/multiselect

### Angular version

17.0.0

### PrimeNG version

17.13.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Steps to reproduce the behavior

https://primeng.org/multiselect open multiselect overlay, run a11y tools
